### ISSUE TITLE
How to reproduce Table 2 from results files

### ISSUE THREAD

**Issue by hnisonoff** [OATML-Markslab/Tranception#12]
Hi, I am trying to reproduce the results in Table 2 from the paper. 

To compute the average spearman for EVE across the Low MSA depth I did the following:
```
df = pd.read_csv('./ProteinGym/Tranception/proteingym/Detailed_performance_files/Substitutions/Spearman/all_models_substitutions_Spearman_DMS_level.csv')
df = df.loc[df.Neff_L_category == 'low']
print(df.EVE_ensemble.mean())
print(df.EVE_single.mean())
0.42358823529411765
0.41711764705882354
```

As you can see, the results are a bit different from the 0.408 reported in Table 2. Thanks!

**Comment by hnisonoff** [comment]
I think I found an explanation in Appendix E.1

**Comment by hnisonoff** [comment]
OK following the appendix now, but I think I may still be doing something wrong.
```
df = pd.read_csv('/home/hunter/projects/recombination/ProteinGym/Tranception/proteingym/Detailed_performance_files/Substitutions/Spearman/all_models_substitutions_Spearman_DMS_level.csv')
df = df.rename(columns={df.columns[0]: 'Dataset'})
df = df.loc[df.Neff_L_category == 'low']
df.groupby('UniProt_ID').EVE_ensemble.mean().mean()
Out [220]:
0.414
```


```
df = pd.read_csv('/home/hunter/projects/recombination/ProteinGym/Tranception/proteingym/Detailed_performance_files/Substitutions/Spearman/all_models_substitutions_Spearman_DMS_level.csv')
df = df.rename(columns={df.columns[0]: 'Dataset'})
df = df.loc[df.Neff_L_category == 'low']
df.groupby('UniProt_ID').EVE_single.mean().mean()
Out [221]:
0.4074
```

**Comment by pascalnotin** [comment]
Hi Hunter,

The last code snippet you shared looks correct to me and matches the most up-to-date results reported on our [website](https://www.proteingym.org/substitutions).

With that said, if you were compare these results with the ones in the paper (Table 2 and others) you will see minor differences: this is due to the [scope extension](https://github.com/OATML-Markslab/Tranception/issues/9) we made over the summer regarding non-focus columns. Numbers in the paper were obtained before that change: the performance numbers for all models were computed on the subset of mutations at focus positions (for fair comparison between all models when EVE & other alignment-based models could only score this subset). 
The performance numbers now reported in the latest repo files or on our website are for all mutants in ProteinGym (since we extended the scope of EVE and other alignment-based models to all mutants).

Kind regards,
Pascal
